nextflow/MaveDB/nextflow.config

```diff
@@ -24,7 +30,10 @@ singularity {
 
 process {
   queue  = 'production'
+  withLabel: bigmem { queue = 'bigmem' }
```

Comment:
You cannot specify queue `bigmem` in slurm - it is automatically assigned based on memory requested. This should be done only in case for LSF.

Reply:
Removed mention to queues for now. We can revisit this in the future. Thanks!